src/server/req_runjob.c

```diff
@@ -301,7 +301,7 @@ req_runjob(struct batch_request *preq)
 	int		  j;
 	char		 *jid;
 	int		  jt;		/* job type */
-	int		  offset;
+	int       js;		/* job state */
```

Comment:
Format this line to align with its previous one.

Reply:
Fixed in rc 15 https://github.com/PBSPro/pbspro/pull/590/commits/bf1558fc183bcfedea1c42d6e44c38c45870d519